groupAndroidPackages.json

```diff
@@ -347,6 +347,13 @@
         "org.threeten:threetenbp",
         "com.jakewharton.threetenabp:threetenabp"
       ]
+    },
+    {
+      "groupName": "KotlinPoet",
```

Comment:
Please sort it alphabetically (between `Kotlin` and `kotlinx.coroutines`).

Reply:
Fixed : https://github.com/hatena/renovate-config/pull/54/commits/c3cb3031b9530fdc270735d2ad477315387e6ffe